requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ flake8==3.8.3
 nose==1.3.7
 splinter>=0.13.0
 -e git+https://github.com/qld-gov-au/ckanext-datarequests@2.1.3-qgov#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-data-qld@3.0.3#egg=ckanext-data_qld
+-e git+https://github.com/qld-gov-au/ckanext-data-qld@develop#egg=ckanext-data_qld
```

Comment:
Why is this ditching the tag? Using specific versions of other extensions helps in detecting backwards compatibility problems.

Reply:
The reason it is updated to the develop branch was for the automation test to pass when the changes from this PR has been merged https://github.com/qld-gov-au/ckanext-data-qld/pull/105 into the develop branch, otherwise the tests would fail because they depended on some new template helpers from ckanext-data-qld.
How would you like us to handle this scenario and future PR's for the project when automation tests require updates from other extensions?

Reply:
Changes should be backward compatible in at least one direction. If `ckanext-data-qld` and `ckanext-data-qld-theme` absolutely must be updated in lockstep and can't be decoupled, that suggests that they ought to be restructured, perhaps moving theme-specific functionality into the theme extension.

Reply:
for time being leave as develop, once we cut a release, move back to fixed version to mimic staging and prod releases.